Drivers/seds/Propulsion.xml

```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PackageFile xmlns="http://www.ccsds.org/schema/sois/seds" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.ccsds.org/schema/sois/seds seds.xsd">
+  <Package name="com/n7space/Propulsion">
+    <DataTypeSet>
+      <EnumeratedDataType name="Propulsion_ThrusterState">
+        <LongDescription>Definitions of thruster states</LongDescription>
+        <IntegerDataEncoding encoding="unsigned" byteOrder="littleEndian" sizeInBits="8" />
+        <EnumerationList>
+          <Enumeration label="Propulsion_ThrusterState_Off" value="0"></Enumeration>
+          <Enumeration label="Propulsion_ThrusterState_BlinkLow" value="1" />
+          <Enumeration label="Propulsion_ThrusterState_BlinkHigh" value="2" />
+          <Enumeration label="Propulsion_ThrusterState_On" value="3" />
+        </EnumerationList>
+      </EnumeratedDataType>
+      <EnumeratedDataType name="Propulsion_ThrusterIndex">
+        <LongDescription>Definitions of thrusters indexes</LongDescription>
+        <IntegerDataEncoding encoding="unsigned" byteOrder="littleEndian" sizeInBits="8" />
+        <EnumerationList>
+          <Enumeration label="Propulsion_ThrusterIndex_0" value="0"></Enumeration>
+          <Enumeration label="Propulsion_ThrusterIndex_1" value="1" />
+          <Enumeration label="Propulsion_ThrusterIndex_2" value="2" />
+          <Enumeration label="Propulsion_ThrusterIndex_3" value="3" />
+        </EnumerationList>
+      </EnumeratedDataType>
+      <ContainerDataType name="Propulsion_Thruster" shortDescription="Single thruster descriptor">
+        <EntryList>
+          <Entry name="mPio" type="com/n7space/PioHwas/PioHwas" />
+          <Entry name="mState" type="Propulsion_ThrusterState" />
+          <Entry name="mCnt" type="com/n7space/Hwas/Byte" />
+        </EntryList>
+      </ContainerDataType>
+      <ContainerDataType name="Propulsion" shortDescription="Propulsion descriptor">
+        <EntryList>
+          <Entry name="mThruster0" type="Propulsion_Thruster" />

```

Comment:
Why use a conteiner, if this could be an array? And then some things could be done in a loop, instead of being copy pasted....